js/addToTask.js

```diff
@@ -0,0 +1,71 @@
+import {
+  createTask,
+  content,
+  list,
+  inputTask,
+  input,
+  addTask,
+  trashInput,
+  today,
+} from "./constants.js"; //named import
+
+export let LIST = [];
+export let id;
+let tarikh;
+function addToDo(toDo, id, done, edit, trash) {
+  if (trash) {
+    return;
+  }
+
+  //Presenting Todays date in task list inside li tag
+  const options = {
+    day: "numeric",
+    month: "numeric",
+    year: "numeric",
+  };
+
+  tarikh = today.toLocaleDateString("en-us", options);
+  const item = `<li class="item">                  
+                  <p class="text">${toDo}</p>
+                  <!-- data presenter div tag with date class  -->
+                  <div class = "date" id = "${id}"> Created At:  ${tarikh} </div>
+                  <img src= "icons/done.svg" job = "complete" id ="${id}"/>
+                  <img src = "icons/edit.svg" job = "edit" id = "${id}"/>
+                  <img src = "icons/delete.svg" job="delete" id="${id}"/>
+                  </li> `;
+
+  const position = "beforeend";
+  list.insertAdjacentHTML(position, item);
+}
+
+createTask.addEventListener("click", function (event) {
```

Comment:
We can denote constants for our event names as well and use that instead.
```suggestion
createTask.addEventListener(CLICK_EVENT, function (event) {
```